### PR TITLE
Feature/1239 add component button

### PR DESCRIPTION
**Pull request by stevenburnell-moj**
Templates tweaks for showing content on component/attributes (accompanies FE work against this ticket in fb-editor repo)

### CODE REVIEW

**Comment by stevenburnell-moj** [comment]
@brenetic
Do we still need these changes or were they incorporated into the updating work you and @njseeto did yesterday?

File: app/views/metadata_presenter/component/_content.html.erb [hunk 1]
```diff
@@ -1 +1 @@
-<%= to_markdown(component.html) %>
+<%= component.html.html_safe %>
```
Comment (brenetic) [inline app/views/metadata_presenter/component/_content.html.erb:1]:
```suggestion
<%= component.content.html_safe %>
```

File: lib/metadata_presenter/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module MetadataPresenter
-  VERSION = '0.26.1'.freeze
+  VERSION = '0.26.2'.freeze
```
Comment (brenetic) [inline lib/metadata_presenter/version.rb:2]: i think this has now gone to 0.27.0

Comment (stevenburnell-moj) [reply lib/metadata_presenter/version.rb]: Looks like current version has already moved on to 0.27.1 so I've made it 0.27.2 for this PR.
See https://github.com/ministryofjustice/fb-metadata-presenter/pull/104/commits/fcf72fd0491cbe505a09b8c173aa7064a9535e6a

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**